internal/download/get.go

```diff
@@ -45,8 +45,12 @@ func httpGet(url string) ([]byte, *failures.Failure) {
 }
 
 func httpGetWithProgress(url string, progress *progress.Progress) ([]byte, *failures.Failure) {
-	logging.Debug("Retrieving url: %s", url)
-	client := retryhttp.NewClient(0 /* 0 = no timeout */, 5)
+	return httpGetWithProgressRetry(url, progress, 0, 3)
```

Comment:
Starting with attempt 0 will result in attempt counts showing as 0-2. It would make more sense in the logs if 0 was 1, and the conditional below was updated to `if attempt <= retries {`. For caution, httpGetWithProgressRetry can check for `<= 0` and bump to 1.